src/CppAst/CppTypeKind.cs

```diff
@@ -49,9 +49,13 @@ public enum CppTypeKind
         /// A template parameter type.
         /// </summary>
         TemplateParameterType,
-        /// <summary>
-        /// An unexposed type.
-        /// </summary>
-        Unexposed,
+		/// <summary>
+		/// A none type template parameter type.
+		/// </summary>
+		NoneTypeTemplateParameterType,
```

Comment:
TemplateParameterNonType

Reply:
Thank you, I have tried to fix these issues and have now committed the change